tests/plenary/ui/mappings_spec.lua

```diff
@@ -1568,4 +1570,29 @@ describe('Mappings', function()
     vim.cmd('norm ,o,a\r')
     assert.are.same('* [#A] Test orgmode', vim.fn.getline(1))
   end)
+
+  it('should refile to headline that matches name exactly', function()
+    local destination_file = helpers.load_file_content({
+      '* foobar',
+      '* baz',
+      '** foo',
+    })
+
+    local source_file = helpers.load_file_content({
+      '* to be refiled',
+      '* not to be refiled',
+    })
+
+    source_file = Files.get_current_file()
+    local item = source_file:get_closest_headline()
+    Capture:refile_to_headline(destination_file, source_file:get_headline_lines(item), item, 'foo')
```

Comment:
```suggestion
    local org = require('orgmode') -- you can move this up outside of tests
    org.instance().capture:refile_to_headline(destination_file, source_file:get_headline_lines(item), item, 'foo')
```

Reply:
When I add this change, `instance` is a nil value field (which is also not callable, but removing the parenthesis makes no difference, still nil)

Reply:
You need to rebase from master branch. I added `instance()` function recently.

Reply:
Thanks, fixed it